.coderabbit.yml

```
@@ -0,0 +1,11 @@
+language: "en-US"
+reviews:
+  profile: "balanced" 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)